Assets/Scripts/PlayerController.cs

```diff
@@ -54,7 +55,15 @@ void Update()
         {
             playerRigidbody.AddForce(Vector2.up * jumpForce, ForceMode2D.Impulse);
         }
+        if (Input.GetKey(KeyCode.J))
```

Comment:
I would change this button to another key. J just seems like a bit of a reach. Unless you are supposed to use two hands.

Reply:
Yes, you are intended to use two hands as there is a second mechanic involving resurrecting the enemy in case they do not go in the right spot.